### PR TITLE
Fix incorrect error message in z_sendmany

### PR DESCRIPTION
**Pull request by bitcartel**
Fixes one of the problems in #1616.

### CODE REVIEW

**Comment by daira** [comment]
ACK modulo comment.

File: src/wallet/asyncrpcoperation_sendmany.cpp [hunk 1]
```diff
@@ -197,11 +197,11 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     CAmount targetAmount = sendAmount + minersFee;
```
Comment (bitcartel) [inline src/wallet/asyncrpcoperation_sendmany.cpp]: Done.  Assert added.

**Comment by bitcartel** [comment]
Review needed for one more ACK.

**Comment by str4d** [comment]
ACK

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**